maintenance/maintenance.html

```diff
@@ -114,37 +114,34 @@ <h4 class="h3-heading mb-4">Let’s protect the world’s largest resource for f
     <footer class="site-footer dark-theme py-5">
       <div class="container">
         <div class="row">
-          <div class="col-md-5">

```

Comment:
This page is not available on review app. 

I updated footer of the maintenance page so links and layout match what we have on main Foundation site.

<img width="1274" alt="image" src="https://user-images.githubusercontent.com/2896608/88248065-bb4e2000-cc54-11ea-8bae-aa7c12cd2f15.png">